libraries/bezier-rs/src/bezier/solvers.rs

```diff
@@ -390,6 +394,15 @@ impl Bezier {
 		.flat_map(|bezier| self.intersections(bezier, None, None))
 		.collect()
 	}
+
+	/// Returns a cubic bezier which joins this and the provided bezier curves.
+	/// The resulting path formed by the Bezier curves is continuous up to the first derivative.
+	/// <iframe frameBorder="0" width="100%" height="325px" src="https://graphite.rs/bezier-rs-demos#bezier/join/solo" title="Join Demo"></iframe>
+	pub fn join(&self, bezier2: &Bezier) -> Bezier {
```

Comment:
I think `other` would be a more traditional name for this kind of code used in conjunction with `self`.